recipes/oatpp/all/conanfile.py

```diff
@@ -73,6 +73,8 @@ def generate(self):
         if is_msvc(self) and Version(self.version) >= "1.3.0":
             tc.variables["OATPP_MSVC_LINK_STATIC_RUNTIME"] = is_msvc_static_runtime(self)
         tc.variables["OATPP_LINK_TEST_LIBRARY"] = self.options.with_test_library
+        if Version(self.version) <= "1.3.0.latest":
```

Comment:
```suggestion
        if Version(self.version) <= "1.3.1":
```

Even not present in CCI, we can be sure 1.3.1 still requires CMake 3.1